src/main/steps/c100-rebuild/child-details/personal-details/content.ts

```diff
@@ -84,13 +86,170 @@ const languages = {
+        },
+      ],
+      parser: body => covertToDateObject('dateOfBirth', body as Record<string, unknown>),
+      validator: (value, formData) =>
+        formData?.isDateOfBirthUnknown !== YesNoEmpty.YES
+          ? areDateFieldsFilledIn(value as CaseDate) ||
+            isDateInputInvalid(value as CaseDate) ||
+            isFutureDate(value as CaseDate)
+          : formData?.isDateOfBirthUnknown === YesNoEmpty.YES
+          ? formData.dateOfBirth.day !== '' || formData.dateOfBirth.month !== '' || formData.dateOfBirth.year !== ''
+            ? 'cannotHaveBothApproxAndExact'
+            : ''
+          : '',
+    },
+    isDateOfBirthUnknown: {
+      type: 'checkboxes',
+      values: [
+        {
+          name: 'isDateOfBirthUnknown',
+          label: l => l.approxCheckboxLabel,
+          selected: isDateOfBirthUnknown === YesNoEmpty.YES,
+          value: YesNoEmpty.YES,
+          subFields: {
+            approxDateOfBirth: {
+              type: 'date',
+              classes: 'govuk-date-input',
+              labelSize: 's',
+              label: l => l.approxDobLabel,
+              values: [
+                {
+                  label: l => l.dateFormat['day'],
+                  name: 'day',
+                  value: approxDateOfBirth!.day,
+                  classes: 'govuk-input--width-2',
+                  attributes: { maxLength: 2, pattern: '[0-9]*', inputMode: 'numeric' },
+                },
+                {
+                  label: l => l.dateFormat['month'],
+                  name: 'month',
+                  value: approxDateOfBirth!.month,
+                  classes: 'govuk-input--width-2',
+                  attributes: { maxLength: 2, pattern: '[0-9]*', inputMode: 'numeric' },
+                },
+                {
+                  label: l => l.dateFormat['year'],
+                  name: 'year',
+                  value: approxDateOfBirth!.year,
+                  classes: 'govuk-input--width-4',
+                  attributes: { maxLength: 4, pattern: '[0-9]*', inputMode: 'numeric' },
+                },
+              ],
+              parser: body => covertToDateObject('approxDateOfBirth', body as Record<string, unknown>),
+              validator: (value, formData) =>
+                formData?.isDateOfBirthUnknown === YesNoEmpty.YES
+                  ? areDateFieldsFilledIn(value as CaseDate) ||
+                    isDateInputInvalid(value as CaseDate) ||
+                    isFutureDate(value as CaseDate)
+                  : '',
+            },
+          },
+        },
+      ],
+    },
+    sex: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      label: l => l.childSexLabel,
+      labelSize: 's',
+      values: [
+        {
+          label: l => l.female,
+          value: Gender.FEMALE,
+        },
+        {
+          label: l => l.male,
+          value: Gender.MALE,
+        },
+        {
+          label: l => l.unspecified,
```

Comment:
Can we rename to 'Other' & enable a textbox to input details when selected instead of Unspecified.